### DISCUSSION TITLE
테스트등록 상태인 솔루션을 설치하려면 어떻게 해야 할까요

### DISCUSSION THREAD

**Discussion by hoksi** [commerce-api-naver/commerce-solutions#77] [묻고 답하기]
![image](https://github.com/user-attachments/assets/5490db41-2e78-45f0-a265-4822245dbd4d)

메뉴얼상으로 대상 솔루션을 생성한 커머스 ID로 로그인하면 설치 가능하다고 되어 있는데 솔루션 생성한 커머스 ID로 로그인후

https://solution.smartstore.naver.com/ko/home

을 살펴봐도  테스트등록 상태인 솔루션이 보이지가 않습니다.

설치를 위해서는 어떻게 하면 좋을까요?

**Comment by commerce-api-naver** [comment]
@hoksi 

안녕하세요 커머스API 권현철입니다.

말씀하신 내용에 대해 답변 드리겠습니다.

커머스솔루션 테스트를 위해서는 커머스솔루션마켓 내에서 솔루션을 관리하는 커머스ID로 로그인을 한다면 'OOO님께만 보이는 솔루션'과 같은 안내 문구가 표시됩니다. 커머스솔루션마켓에서 커머스ID로 로그인 후에도 동일하게 표시되지 않는지, 마켓 콘텐츠를 정상적으로 입력했는지 등 확인 가능하실까요?

감사합니다.

**Comment by hoksi** [comment]
![image](https://github.com/user-attachments/assets/a22b1ef4-cbff-4cfc-9943-5c5c8365b620)

커머스 솔루션 마켓에 동일한 ID로 로그인하여도 'OOO님께만 보이는 솔루션'에 표시가 되지 않습니다.

![image](https://github.com/user-attachments/assets/4fe22140-41ea-4c4d-9d67-ff1d254847f2)

그리드상에 보이는 정보로는 마켓 콘텐츠가 정상 입력된 것으로 보입니다.

정상 입력 여부를 확인할 다른 방법이 있을까요?

**Reply by commerce-api-naver** [reply]
@hoksi 

안녕하세요. 커머스API 권현철입니다.

커머스솔루션마켓에 테스트 솔루션이 노출되기 위해서는 솔루션과 연동된 계정이 스마트스토어의 계정 주매니저 이상의 매니저 권한이 있어야 가능합니다. 로그인 한 계정이 계정 주매니저 이상의 권한을 가진 계정인지 확인 부탁드리며 권한 없거나 낮은 경우 [스마트스토어센터 > 판매자 정보 > 판매자 관리 > 매니저 관리]에서 권한 수정 및 매니저 초대 부탁드립니다.

감사합니다.

**Comment by hoksi** [comment]
빠른 답변 감사합니다.

[스마트스토어센터 > 판매자 정보 > 판매자 관리 > 매니저 관리] 에서 권한 확인 결과 "통합 매니저"로 설정되어 있는것을 확인 하였습니다.
![image](https://github.com/user-attachments/assets/c876a3b0-4047-48cc-8abc-c59e4e189e27)

다른 계정을 등록하여 테스트를 진행하여야 할까요?

**Reply by commerce-api-naver** [reply]
@hoksi 

안녕하세요 커머스API 권현철입니다.

이전 답변 간 수정이 필요하여 재답변 드립니다.
> 커머스솔루션마켓에 테스트 솔루션이 노출되기 위해서는...
해당 내용에 대해서 추가 확인한 결과 테스트등록된 솔루션의 경우 커머스솔루션마켓 홈에서 확인이 불가능한 것으로 확인되었습니다.
테스트를 위해서는 [커머스API센터 > 솔루션 > 내 솔루션 > 마켓 노출 > 노출]링크 클릭을 통해서 진입이 가능합니다.

진입 시에는 앞서 언급 드린 내용처럼 솔루션과 연동된 계정이 스마트스토어의 계정 주매니저 이상의 매니저 권한이 있어야 가능합니다.
위 진입 방법을 통해 정상적으로 테스트 가능할지 확인 부탁드립니다.

답변 간 혼선을 드려 죄송합니다.

감사합니다.

**Comment by hoksi** [comment]
![image](https://github.com/user-attachments/assets/8de08f35-b955-4bcf-ac07-93f221b306c4)

그리드의 링크를 통하여 접근 가능함을 확인하였습니다.

감사합니다.
